GlobalQuakeClient/src/main/java/globalquake/ui/globalquake/EarthquakeListPanel.java

```diff
@@ -109,6 +138,23 @@ public void mouseMoved(MouseEvent e) {
         });
     }
 
+    /**
+     * @author Chris Eberle
+     * @param UUID of an archived quake
+     * @return boolean representing whether a quakepanel is open
+     * 
+     * Compares the inputed id with all id's stored in openedquakes list
+    */
+    private boolean isOpened(UUID id) {
+        // loop through openedquakes
+        for( UUID i: openedQuakes) {
+            if( i == id) {
+                return true;
```

Comment:
Would it be possible to use the ArrayList builtin `indexOf(id)` or `contains(id)` instead of looping over the array?

Reply:
Just replaced my isOpened method with contains! Thanks for the tip, didn't know this method existed!!